devito/passes/iet/misc.py

```diff
@@ -58,11 +58,9 @@ def hoist_prodders(iet):
 @iet_pass
 def relax_incr_dimensions(iet, **kwargs):
     """
-    Recast Iterations over IncrDimensions as ElementalFunctions; insert
-    ElementalCalls to iterate over the "main" and "remainder" regions induced
-    by the IncrDimensions.
+    Recast Iteration bounds using min/max conditions to iterate over the
+    domain. This function sets the limits of a tree's Iterations.
```

Comment:
it doesn't really "set" anything -- it's not changing any state. It rather builds new Iteration trees such that ...